README.md

```diff
@@ -176,5 +177,4 @@ Sample code
 
 ## Future work
 
-Since there is no alternate api for setColorFilter and getOpacity in HMOS platform, so those functions is currently not supported. 
-This library draws the image and text directly on the provided canvas, so we cannot use TextWithImageDrawable as an input to BaseCombinedDrawable and width and height of the image should be managed properly
\ No newline at end of file
+Since there is no alternate api for setColorFilter and getOpacity in HMOS platform, so those functions is currently not supported. 
```

Comment:
Modify Future work section to this - "Since there are no alternate APIs for setColorFilter and getOpacity in HarmonyOS platform, these functions are currently not supported in this library. Once the platform includes the support, the above API functionality can be integrated in this library."